src/Molecules/UserSelect/UserSelect.stories.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react'
+
+import { action } from '@storybook/addon-actions'
+import {  } from '@storybook/addon-knobs'
+
+import UserSelect, { User } from './index'
+import { TransProvider } from '../../Atoms/Trans/index'
+
+import markdown from './README.md'
+
+export default {
+  title: 'Molecules/UserSelect',
+}
+
+const translations = {
+  global: {
+    open:  'Open jedis',
+    close: 'Close jedis',
+    clear: 'Clear jedi',
+  },
+}
+
+export const userSelect = () => {
+  const handleError = ({ error }) => window.alert(error)
+
+  return (
+    <TransProvider value={translations}>
+      {
+        [
+          {
+            id:       'jedi',
+            label:    'Jedi and not jedi',
+            useState: useState(1),
+          },
+          {
+            id:       'jedis',
+            label:    'Jedis and not jedis',
+            useState: useState([1]),
+          },
+        ].map(({ id, label, useState: [state, setState] }, key) => (
+          <UserSelect
+            key={key}
+            id={id}
+            label={label}
+            values={state}
+            onChange={value => action('change')(value) || setState(value)}
+            onError={error => action('error')(error) || handleError(error)}
```

Comment:
why not with `&&` ? is there any difference ?

Reply:
Yup, `action` returns a falsy value

Reply:
damn